metamorphic/ops.go

```diff
@@ -1988,7 +1988,7 @@ func (r *replicateOp) runSharedReplicate(
 		return
 	}
 
-	_, err = dest.IngestAndExcise([]string{sstPath}, sharedSSTs, nil /* external */, pebble.KeyRange{Start: r.start, End: r.end})
+	_, err = dest.IngestAndExcise([]string{sstPath}, sharedSSTs, nil /* external */, pebble.KeyRange{Start: r.start, End: r.end}, false)
```

Comment:
Let's open an issue to toggle this to true for some metamorphic runs (and if we're toggling this to true, we can throw in the relevant dels inside the ssts)

Reply:
https://github.com/cockroachdb/pebble/issues/3432